desdeo_emo/EAs/MOEAD.py

```diff
@@ -0,0 +1,164 @@
+from typing import Dict
+
+from numpy import argsort
```

Comment:
Just import numpy as np. It's easier to understand in the code that "np.argsort" comes from numpy that way. It will also reduce the number of import statements.